src/prommis/uky/uky_flowsheet.py

```diff
@@ -2835,15 +2912,6 @@ def add_costing(m):
     # fix costing vars that shouldn't change
     m.fs.precipitate.fix()
 
-    # Initialize costing
-    QGESSCostingData.costing_initialization(m.fs.costing)
-    QGESSCostingData.initialize_fixed_OM_costs(m.fs.costing)
-    QGESSCostingData.initialize_variable_OM_costs(m.fs.costing)
-
-    # Solve costing
-    solver = get_solver()
-    solver.solve(m, tee=True)
```

Comment:
Why is this being deleted?

Reply:
I think separating this out is a good idea in terms of being able to isolate the individual components for testing purposes. But now the costing results being displayed when running the flowsheet aren't accurate, right? Wouldn't we need to add these lines after `add_costing` in the main function?

Reply:
You're right, I missed that. I'll update it now, thanks.